lunatrace/bsl/frontend/src/components/finding/VulnerablePackageList.tsx

```diff
@@ -11,25 +11,30 @@
  * limitations under the License.
  *
  */
-import { groupByPackage, severityOrder } from '@lunatrace/lunatrace-common';
-import React, { useState } from 'react';
+import { groupByPackage, severityOrder } from '@lunatrace/lunatrace-common/build/main';
+import React, { ChangeEvent, useState } from 'react';
 import { Col, Container, Dropdown, Row } from 'react-bootstrap';
 
 import { VulnerablePackageItem } from './VulnerablePackageItem';
 import { Finding } from './types';
 
+export interface QuickViewProps {
```

Comment:
nice and i almost did this when i wrote it.  Since we use this all over, why not move it into the types file.